assets/admin/editor-wizard/wizard.js

```diff
@@ -34,20 +36,25 @@ const Wizard = ( { steps, onCompletion } ) => {
 				/>
 				<div className="sensei-editor-wizard__footer">
 					<div className="sensei-editor-wizard__progress">
-						Step { currentStepNumber + 1 } of { steps.length }
+						{ sprintf(
+							// translators: %1$d Current step number, %2$d Number of steps.
+							__( 'Step %1$d of %2$d', 'sensei-lms' ),
+							currentStepNumber + 1,
+							steps.length
+						) }
 					</div>
 					{ CurrentStep.Actions && (
 						<div className="sensei-editor-wizard__actions">
 							<CurrentStep.Actions
 								data={ data }
 								goToNextStep={ goToNextStep }
+								skipWizard={ skipWizard }
 							/>
 						</div>
 					) }
 				</div>
 			</div>
-		) ) ||
-		null
+		)
```

Comment:
I remember needing to add this `|| null` because I was getting an error that a render should return something always. Why it is not needed anymore?

Reply:
Ah, I got it! If `steps[ currentStepNumber ]` is `undefined` it would fire an error. But if we return `null` we also have a weird behavior with the modal open, but only with the overlay.

Thinking a little more, I don't think we need this check since `goToNextStep` will only increment if we have steps. I removed it for now in this commit: https://github.com/Automattic/sensei/pull/5179/commits/946f50d20a7844c927000d3693037f0b0b41d583, but let me know if I'm missing anything. 😉

Reply:
I also removed the conditional in the other case where steps should always be an array.